start.aws.batch.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
```

Comment:
We now have 4 scripts: start.lsf, start.slurm, start.gls and start.aws. There are a lot of repeated implementation in the 4 scripts. This can become a maintenance nightmare. Again, we need to get together to figure out how to implement start.xxx and run.nextflow together.

My original design was not good thus we are running into this maintenability issue. If anyone to blame, it's me. Let's figure out how to make it better together.